usaspending_api/transactions/tests/integration/test_assistance_etl.py

```diff
@@ -0,0 +1,268 @@
+(E'2018-04-17 17:36:26.770692',E'2018-04-17 17:36:26.770699',81927035,E'20180228',NULL,E'11',NULL,E'MULTIPLE RECIPIENTS',NULL,E'036',NULL,E'3620',NULL,E'NON',E'X',E'64.013',NULL,NULL,NULL,32816,NULL,E'036',NULL,E'3600',NULL,NULL,NULL,E'USA',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'GA**069',E'12',E'USA',NULL,NULL,1,NULL,E'18050413.0130353',E'12',E'32816',E'Veterans Prosthetic Appliances',E'Department of Veterans Affairs (VA)',E'Under Secretary for Health/Veterans Health Administration',E'Department of Veterans Affairs (VA)',E'Department of Veterans Affairs',NULL,E'Coffee',E'Georgia',NULL,NULL,E'069',E'Coffee',E'GA',E'Georgia',E'2018-04-17 17:36:26.769348',E'3620_-none-_18050413.0130353_64.013_-none-',TRUE,NULL,NULL,NULL,NULL,E'UNITED STATES',E'UNITED STATES',E'069',9643,E'GA',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'ASST_AGG_18050413.0130353_3620',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'County-wide'),
+(E'2018-11-28 20:12:41.054538',E'2018-11-28 20:12:41.054543',89126414,E'20181003',E'A',E'10',E'PRICE LOSS COVERAGE PROGRAM',E'Redacted due to PII',NULL,E'012',E'12346T',E'12D2',NULL,E'NON',E'P',E'10.112',NULL,NULL,E'12FA00PY57991084',7827,NULL,E'012',E'12346T',E'12D2',NULL,NULL,NULL,E'USA',NULL,NULL,NULL,E'59427',NULL,0,NULL,E'20181003',E'20181003',E'MT18775',E'00',E'USA',NULL,E'59427',3,E'SAI EXEMPT',NULL,E'00',E'7827.00',E'Price Loss Coverage',E'Department of Agriculture (USDA)',E'Farm Service Agency',E'Department of Agriculture (USDA)',E'Farm Service Agency',NULL,E'Glacier',E'Montana',E'CUT BANK',E'CUT BANK',E'035',E'Glacier',E'MT',E'Montana',E'2018-11-28 20:12:41.053555',E'12D2_12FA00PY57991084_-none-_10.112_-none-',TRUE,E'Deputy Administrator Farm Programs',E'Deputy Administrator Farm Programs',E'18775',NULL,E'UNITED STATES',E'UNITED STATES',E'035',14901,NULL,E'59427',NULL,E'{{individuals}}',E'New',E'direct payment with unrestricted use (retirement, pension, veterans benefits, etc.) (D)',E'Not Recovery Act',E'Individual',NULL,E'Non-Aggregate Record to an Individual Recipient (PII-Redacted)',NULL,NULL,E'ASST_NON_12FA00PY57991084_12D2',NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,E'Single ZIP Code'),
+(E'2018-11-28 20:12:41.143701',E'2018-11-28 20:12:41.143706',89126418,E'20181003',E'A',E'10',E'AGRICULTURAL RISK COVERAGE PROG - COUNTY',E'Redacted due to PII',NULL,E'012',E'12346T',E'12D2',NULL,E'NON',E'P',E'10.112',NULL,NULL,E'12FA00PY57991082',1468,NULL,E'012',E'12346T',E'12D2',NULL,NULL,NULL,E'USA',NULL,NULL,NULL,E'57366',NULL,0,NULL,E'20181003',E'20181003',E'SD48460',E'00',E'USA',NULL,E'57366',3,E'SAI EXEMPT',NULL,E'00',E'1468.00',E'Price Loss Coverage',E'Department of Agriculture (USDA)',E'Farm Service Agency',E'Department of Agriculture (USDA)',E'Farm Service Agency',NULL,E'Hutchinson',E'South Dakota',E'PARKSTON',E'PARKSTON',E'067',E'Hutchinson',E'SD',E'South Dakota',E'2018-11-28 20:12:41.14262',E'12D2_12
```

Comment:
Don't think we need this test since it already exists as part of `test_broker_integration.py`.

Reply:
Correct. And I can remove it from both of the new test files.

It takes a fraction of a second to run and is useful for troubleshooting.